Ali | Amir / Storybook for components (#2009)
binary-com/deriv-app: packages/components/stories/themed-scrollbars/README.md

# ThemedScrollbar component

> Wrap an element to have a custome scrollbar for it.

Reviewer: ```suggestion
Wrap an element to have a custom scrollbar for it.
```

> Wrap an element to have a customed scrollbar for it.

Reviewer 2: ```suggestion
Wrap an element to have a custome scrollbar for it.
```

| height                         | {string}               | 100%               | Scroll Component height                                                                                                  |
| width                          | {string}               | none               | Scroll Component width                                                                                                   |
| autohide                       | {boolean}              | true               | Hide scrollbar if component is not hovered                                                                               |
| is\_bypassed                   | {boolean}              | null               | Set it to 'true' if you want to disable the custom scrollbar component                                                   |
| is\_only\_horizontal           | {boolean}              | null               | Set it to 'true' if you want to disable the vertical scrollbar                                                           |

> | is\_only\_horizontal\_overlay  | {boolean}              | null               | It's same as 'is\_only\_horizontal' but it use 'overflow-x: overlay' style.                                              |

Reviewer 2: ```suggestion
| is\_only\_horizontal\_overlay  | {boolean}              | null               | It's same as 'is\_only\_horizontal' but using 'overflow-x: overlay' style.                                              |
```

> | has\_horizontal                | {boolean}              | null               | If you want to have vertical and horizontal scrollbar set it to 'true'.                                                  |

Reviewer: ```suggestion
| has\_horizontal                | {boolean}              | null               | If you want to have vertical and horizontal scrollbars set it to 'true'.                                                  |
```

| onScroll                       | {function}             | null               | Call back function for scrolling event.                                                                                  |

> | refSetter                      | {object} React ref     | null               | Ref object that component uses it to handle the hover event.                                                             |

Reviewer: ```suggestion
| refSetter                      | {object} React ref     | null               | Ref object that component uses to handle the hover event.                                                             |
```
